### PR TITLE
Bug/cer 57 make person not found throw 404 not found

### PR DESCRIPTION
**Pull request by jtorano-va**
https://vasdvp.atlassian.net/browse/CER-57

### CODE REVIEW

**Comment by dtournour-va** [comment]
This change could break CCE.  So let us know

File: hand-of-queen-elizabeth/src/main/java/gov/va/api/health/queenelizabeth/ee/Eligibilities.java [hunk 1]
```diff
@@ -22,22 +22,29 @@ public MissingIcnValue(SoapMessageGenerator soapMessageGenerator) {
...
 
     public RequestFailed(SoapMessageGenerator soapMessageGenerator, String message) {
       super(
-          soapMessageGenerator.createGetEeSummarySoapRequest().toString() + " Reason: " + message);
+          soapMessageGenerator.createGetEeSummarySoapRequest().toString() + " Reason1: " + message);
     }
 
     public RequestFailed(SOAPMessage soapRequestMessage, String message) {
-      super(soapRequestMessage.toString() + " Reason: " + message);
+      super(soapRequestMessage.toString() + " Reason2: " + message);
```
Comment (jblefkove-va) [inline hand-of-queen-elizabeth/src/main/java/gov/va/api/health/queenelizabeth/ee/Eligibilities.java]: Why `Reason1` vs `Reason2` ?

Comment (jtorano-va) [reply hand-of-queen-elizabeth/src/main/java/gov/va/api/health/queenelizabeth/ee/Eligibilities.java]: I mistakenly checked those changes in that I was using for at local test. I have removed them.

File: hand-of-queen-elizabeth/src/test/java/gov/va/api/health/queenelizabeth/ee/impl/XmlResponseValidatorTest.java [hunk 2]
```diff
@@ -53,6 +55,25 @@ public void requestFailedForEeFault() {
...
+    Element faultString = response.createElement("faultstring");
+    Text personNotFound = response.createTextNode("PERSON_NOT_FOUND");
+    faultString.appendChild(personNotFound);
+    fault.appendChild(faultString);
+    XmlResponseValidator xmlResponseValidator =
+        XmlResponseValidator.builder()
+            .soapMessageGenerator(soapMessageGenerator())
+            .response(response)
+            .build();
+    xmlResponseValidator.validate();
```
Comment (jblefkove-va) [inline hand-of-queen-elizabeth/src/test/java/gov/va/api/health/queenelizabeth/ee/impl/XmlResponseValidatorTest.java]: If you add a sample response to `src/test/resources` for PERSON_NOT_FOUND, you should be able to follow the model of the other tests:

```
  @Test(expected = Eligibilities.PersonNotFound.class)
  public void requestPersonNotFound() {
    parse(Samples.create().personNotFound());
  }
```

**Pull request merged**